plugins/org.komodo.eclipse.workspace.ui/src/org/komodo/eclipse/workspace/ui/editor/KWorkspaceViewerEditorPage.java

```diff
@@ -0,0 +1,384 @@
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Display;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.ui.forms.widgets.ExpandableComposite;
+import org.eclipse.ui.forms.widgets.FormToolkit;
+import org.eclipse.ui.forms.widgets.Section;
+import org.komodo.eclipse.workspace.CoreConstants;
+import org.komodo.eclipse.workspace.CorePlugin;
+import org.komodo.eclipse.workspace.ui.UiConstants;
+import org.komodo.eclipse.workspace.ui.actions.NewObjectAction;
+import org.komodo.eclipse.workspace.ui.actions.NewObjectActionFactory;
+import org.komodo.eclipse.workspace.ui.editor.panels.IKObjectChangeListener;
+import org.komodo.eclipse.workspace.ui.editor.panels.ObjectChangedEvent;
+import org.komodo.eclipse.workspace.ui.editor.panels.ObjectEditorPanel;
+import org.komodo.eclipse.workspace.ui.editor.panels.relational.ColumnEditorPanel;
+import org.komodo.eclipse.workspace.ui.editor.panels.relational.ModelEditorPanel;
+import org.komodo.eclipse.workspace.ui.editor.panels.relational.ParameterEditorPanel;
+import org.komodo.eclipse.workspace.ui.editor.panels.relational.ProcedureEditorPanel;
+import org.komodo.eclipse.workspace.ui.editor.panels.relational.TableEditorPanel;
+import org.komodo.eclipse.workspace.ui.editor.panels.relational.ViewEditorPanel;
+import org.komodo.eclipse.workspace.ui.editor.panels.relational.VirtualProcedureEditorPanel;
+import org.komodo.eclipse.workspace.ui.editor.panels.relational.util.AddNameObjectDialog;
+import org.komodo.eclipse.workspace.ui.editor.panels.vdb.DataPolicyEditorPanel;
+import org.komodo.eclipse.workspace.ui.editor.panels.vdb.DataSourceEditorPanel;
+import org.komodo.eclipse.workspace.ui.editor.panels.vdb.FileEntryEditorPanel;
+import org.komodo.eclipse.workspace.ui.editor.panels.vdb.PermissionEditorPanel;
+import org.komodo.eclipse.workspace.ui.editor.panels.vdb.TranslatorEditorPanel;
+import org.komodo.eclipse.workspace.ui.editor.panels.vdb.VdbEditorPanel;
+import org.komodo.eclipse.workspace.ui.editor.panels.vdb.VdbImportEditorPanel;
+import org.komodo.eclipse.workspace.ui.explorer.WorkspaceTreeViewer;
+import org.komodo.eclipse.workspace.util.RelationalIdentifier;
+import org.komodo.relational.model.Model;
+import org.komodo.relational.model.Table;
+import org.komodo.relational.vdb.Vdb;
+import org.komodo.relational.workspace.WorkspaceManager;
+import org.komodo.spi.KException;
+import org.komodo.spi.repository.KomodoObject;
+import org.komodo.spi.repository.Repository;
+import org.komodo.spi.repository.Repository.UnitOfWork;
+import org.modeshape.sequencer.ddl.StandardDdlLexicon;
+import org.modeshape.sequencer.ddl.dialect.teiid.TeiidDdlLexicon;
+import org.modeshape.sequencer.ddl.dialect.teiid.TeiidDdlLexicon.CreateTable;
+
+public class KWorkspaceViewerEditorPage extends KWorkspaceEditorPage implements IKObjectChangeListener  {
```

Comment:
While we havent actually published the Komodo formatting guidelines, we did decide early on that interfaces would not start with "I".